waku/v2/node/waku_node.nim

```diff
@@ -401,6 +401,7 @@ proc registerRelayDefaultHandler(node: WakuNode, topic: PubsubTopic) =
 
   proc traceHandler(topic: PubsubTopic, data: seq[byte]) {.async, gcsafe.} =
     trace "waku.relay received",
+      peerId=node.switch.peerInfo.peerId,
```

Comment:
I see that this leaks the "switch" implementation detail. Could you add a `proc peerId(): PeerId =` to `WakuNode`? 
 
IIRC, I already added a `peerInfo` proc. It would be as simple as adding a `PeerId` accessor, so we don't leak the "switch" in different places.

Reply:
Oh, yeah makes sense.

Done in [6429b90](https://github.com/waku-org/nwaku/pull/1574/commits/6429b903bbc393ce4e7dd4d30a41f5878b42e212)